iroh-rpc-client/src/network.rs

```diff
@@ -270,23 +271,39 @@ impl P2pClient {
     }
 
     #[tracing::instrument(skip(self))]
-    pub async fn check(&self) -> StatusRow {
-        let status: ServiceStatus = self
-            .version()
-            .await
-            .map(|_| ServiceStatus::Serving)
-            .unwrap_or_else(|_e| ServiceStatus::Unknown);
-        StatusRow {
+    pub async fn check(&self) -> ServiceStatus {
+        let (status, version) = match self.version().await {
+            Ok(version) => (StatusType::Serving, version),
+            Err(_) => (StatusType::Down, String::new()),
+        };
+        ServiceStatus {
             name: "p2p",
-            number: 2,
+            number: 1,
```

Comment:
Do we need this number at all? It seems it was meant to be the column or row for the service. But we are doing that with the name now, right?

Reply:
It's a holdover, preemptive work for when we have more than one of `Service`. It's unnecessary right now & I'm planning to do a round to remove it from the response, but keep it in the `iroh status` output.

Reply:
removed `number` field